### PR TITLE
Pr 289

### PR DESCRIPTION
**Pull request by stefanmeissner**
Please have a look if this is sufficient.

Don't merge my artifact id's. I changed them for testing only.

### CODE REVIEW

File: jtwig-core/src/main/java/org/jtwig/util/TypeUtil.java [hunk 1]
```diff
@@ -207,13 +223,13 @@ public static boolean isNumeric(Object obj) {
...
-            return b == null || (isBoolean(b) && !toBoolean(b));
+            return b == null || isEmptyString(b) || (isBoolean(b) && !toBoolean(b));
         }
         if (b == null) {
-            return isBoolean(a) && !toBoolean(a);
+            return isEmptyString(a) || isBoolean(a) && !toBoolean(a);
         }
         if (isBoolean(a) || isBoolean(b)) {
-            return toBoolean(a) == toBoolean(b);
+            return Objects.equals(toBoolean(a), toBoolean(b));
```
Comment (thomas-p-wilson) [inline jtwig-core/src/main/java/org/jtwig/util/TypeUtil.java]: Good catch on this one! I forgot that I was comparing wrappers rather than primitives.

**Comment by thomas-p-wilson** [comment]
:+1: If you have nothing else for this PR, I'll rebase it to remove the artifact naming changes and pull.

File: jtwig-core/src/main/java/org/jtwig/util/TypeUtil.java (hunk 1 above)
Comment (stefanmeissner) [inline jtwig-core/src/main/java/org/jtwig/util/TypeUtil.java]: I guess there are several other things to change. I'll let FindBugs have a run or maybe you know some other tools to check for clean code.

**Comment by stefanmeissner** [comment]
Should be fine to resolve the #289 issue.

**Comment by thomas-p-wilson** [comment]
Manually merged.

**Pull request closed without merging**